test/rsm/test.ts

```diff
@@ -1,44 +1,36 @@
 // See README.md for prerequisites for this to run
 
-const { Orchestrator, Config } = require('../../src')
-
-const testDna = Config.dna("test.dna.gz")
-
-const config = Config.gen({
-  tester: testDna,
-})
+import { Orchestrator, Config, InstallHapps } from "../../src"
+import { ScenarioApi } from "../../src/api"
+import path from 'path'
 
 const orchestrator = new Orchestrator()
 
-orchestrator.registerScenario('list dnas', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dnas = await antony.admin().listDnas()
-  console.log('dnas', dnas)
-
-  t.equal(dnas.length, 1)
-})
-
-orchestrator.registerScenario('call zome', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const result = await antony.call('tester', 'foo', 'foo', { anything: 'goes' })
-  console.log('result', result)
-
-  t.equal(result, 'foo')
-})
-
-orchestrator.registerScenario('state dump', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dump = await antony.stateDump('tester')
-  t.equal(dump.length, 3)
-  t.ok(typeof dump[0].element === 'object')
-  t.ok(typeof dump[1].element === 'object')
-  t.ok(typeof dump[2].element === 'object')
+const conductor1Config = Config.gen()
+const conductor2Config = Config.gen()
+
+const conductor2Apps: InstallHapps = [
+  // agent 1
+  [
+    // dna 1
+    path.join(__dirname, 'test.dna.gz')//,
+    // dna 2...
+  ],
+  // agent 2
+  [
+    // dna 1
+    path.join(__dirname, 'test.dna.gz')//,
+    // dna 2...
+  ]
+]
+
+orchestrator.registerScenario('list dnas', async (s: ScenarioApi, t) => {
+  const [[antonyConductor, [[antony1TestCell], [antony2TestCell]]]] = await s.players(
+    [[conductor2Config, conductor2Apps]]
+  )
```

Comment:
I'm personally cool with this lispy syntax, but it's not very idiomatic JS and may not be so readable for JS devs. I wonder if there's anything we can do to make it a bit more idiomatic. I don't have any ideas at the moment.

Reply:
Yeah it does feel a bit too "deep" ... 
Thinking about how it could be possible to shallow it out.

I had an idea before of taking either arrays or single objects/strings to provide a simpler case, but typescript didn't like th variability of destructuring to a union type, with no type safety.

Reply:
my latest changes provide some new thinking on this. main point is that I think "initialization" was overloaded and that's why we were hitting up against ugly results from calling `s.players`